src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSJobManager.groovy

```diff
@@ -189,7 +189,7 @@ class PBSJobManager extends ClusterJobManager<PBSCommand> {
         StringBuilder sb = new StringBuilder()
 
         if (resourceSet.isMemSet()) sb << " -l mem=" << resourceSet.getMem().toString(BufferUnit.M)
-        if (resourceSet.isWalltimeSet()) sb << " -l walltime=" << durationToPbsWallTime(resourceSet.getWalltime())
+        if (resourceSet.isWalltimeSet()) sb << " -l walltime=" << resourceSet.getWalltime()
```

Comment:
When we use TimeUnit there is no conversion method needed for PBS, but for LSF and SLURM.